src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
```

Comment:
The variable name 'formatedCSSData' is not descriptive enough and could be improved for better readability. Consider renaming it to something more specific like 'styleObject' or 'cssPropertiesObject' to clearly represent its purpose.